contrib/fwupd.spec.in

```diff
@@ -269,6 +269,11 @@ mkdir -p --mode=0700 $RPM_BUILD_ROOT%{_localstatedir}/lib/fwupd/gnupg
 %post
 %systemd_post fwupd.service
 
+# change vendor-installed remotes to use the default keyring type
+for fn in /etc/fwupd/remotes.d/*.conf;
+    do sed -i 's/Keyring=gpg/#Keyring=pkcs/g' "$fn";
+done
```

Comment:
I wonder if we also want this in other packaging types too?  Or possibly it should be code in the daemon to do it?

Reply:
It can't be in the Daemon as in the snap case the Daemon is the old one, not the new one.